eng/common/TestResources/New-TestResources.ps1

```diff
@@ -527,6 +527,8 @@ try {
     if ($CI -and $Environment -eq 'AzureCloud' -and $env:PoolSubnet) {
         $templateParameters.Add('azsdkPipelineSubnetList', @($env:PoolSubnet))
     }
+    # Some arm/bicep templates may want to change deployment settings (e.g. local auth) in sandboxed TME tenants
+    $templateParameters.Add('tenantIsTME', ($context.Tenant.Name -like '*TME*'))
```

Comment:
Instead of being sub based should this be feature based? I.e. supportsLocalAuth or something like that?

Reply:
If possible I would prefer downstream arm templates to be not care about the tenant, so yes.

I wonder if we want to do feature based, or some sort of security level/category? I worry enumerating all the features as more and more parameters could be tedious, vs. something like `disableBroadAccess`.

Reply:
WE could be broad with our features like SupportsSafeSecretStandard or something along those lines. Either way, I like it more than sub or tenant checking. The main question is how we would make a smart default so that both local and pipeline testing works.